kolibri/core/assets/src/core-global.styl

```diff
@@ -106,3 +106,8 @@ button::-moz-focus-inner
   padding: 0
   position: absolute
   width: 1px
+
+.description
+  margin-top: 2em
+  margin-bottom: 1.4em
+  line-height: 1.8em
```

Comment:
It would also be best to give it a slightly more descriptive/unique name, like `page-description`

Basically, we need to be careful about global styles because it's likely that someone might come along and make a locally scoped class named `description`, and this class will conflict with it.